src/test/java/com/snowplowanalytics/snowplow/tracker/SubjectTest.java

```diff
@@ -92,6 +92,13 @@ public void testSetNuid() throws Exception {
         assertEquals("nuid", subject.getSubject().get("tnuid"));
     }
 
+    @Test
+    public void testSetSid() throws Exception {
```

Comment:
Nitpicky: It's just a test but it's a bit cryptic. I see you followed the same style that was already there, which is good in general, but since we test `setDomainSessionId` I would just call this test `testSetDomainSessionId`. I would update the other two tests (`testSetNuid`, `testSetDuid`) too. Just to make the test name a bit more self-explanatory. It's hard to remember all the keys of the atomic table! :D